Datadog.Trace.sln

```diff
@@ -205,6 +205,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AssemblyLoad.FileNotFoundEx
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docker", "docker", "{6ABAD006-E206-488E-ACA1-1AA73F9B5146}"
 	ProjectSection(SolutionItems) = preProject
+		build\docker\build.arm64.sh = build\docker\build.arm64.sh
```

Comment:
Unrelated, but Visual Studio keeps moving this line up to keep them in alphabetical order.